zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/client/ZakenApiClient.kt

```diff
@@ -448,6 +456,127 @@ class ZakenApiClient(
         return result?.body!!
     }
 
+    fun createZaakeigenschap(
+        authentication: ZakenApiAuthentication,
+        baseUrl: URI,
+        request: CreateZaakeigenschapRequest,
+    ): ZaakeigenschapResponse {
+        validateUrlHost(baseUrl, request.zaak)
+        val result = webclientBuilder
+            .clone()
+            .filter(authentication)
+            .build()
```

Comment:
Can we make a private method for this part? It's being repeated over and over.

Reply:
added